client/wait.go

```diff
@@ -0,0 +1,248 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"github.com/zrepl/zrepl/cli"
+	"github.com/zrepl/zrepl/config"
+	"github.com/zrepl/zrepl/daemon"
+	"github.com/zrepl/zrepl/daemon/job"
+	"github.com/zrepl/zrepl/daemon/pruner"
+	"github.com/zrepl/zrepl/daemon/snapper"
+	"github.com/zrepl/zrepl/replication/report"
+)
+
+type wui struct {
+	//lock      sync.Mutex //For report and error
+	report map[string]*job.Status
+
+	jobName string
+}
+
+var WaitCmd = &cli.Subcommand{
+	Use:   "wait JOB",
+	Short: "block and wait until the specified job is done or has failed",
+	Run: func(ctx context.Context, subcommand *cli.Subcommand, args []string) error {
+		return runWaitCmd(subcommand.Config(), args)
+	},
+}
+
+// adapted from status.go
```

Comment:
perhaps we should re-use the `client` from the status-v2 work https://github.com/zrepl/zrepl/pull/297

Reply:
I'll look into it :slightly_smiling_face: